### PR TITLE
Added simple cmake-python build system, main.cpp and gitignore

### PR DESCRIPTION
**Pull request by BoraRider**

### CODE REVIEW

File: Cpp/build.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import os
```
Comment (ProrokWielki) [inline Cpp/build.py]: delete

File: Cpp/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# the project name
+project(MapChunkAnalyzer)
+
+set(SOURCE_DIR source)
+set(SOURCES ${SOURCE_DIR}/main.cpp)
+
+include_directories(${SOURCE_DIR})
+
+add_executable(${CMAKE_PROJECT_NAME} ${SOURCES})
```
Comment (ProrokWielki) [inline Cpp/CMakeLists.txt]: move to CMakeLists.txt in source

File: Cpp/build.py (hunk 1 above)
Comment (BoraRider) [reply Cpp/build.py]: done

File: Cpp/CMakeLists.txt (hunk 2 above)
Comment (BoraRider) [reply Cpp/CMakeLists.txt]: done

**Review by ProrokWielki — APPROVED** [review]

**Pull request merged**